### PR TITLE
[chore] hide floating actions on mobile resolutions

### PR DESCRIPTION
**Pull request by Kharonus**

### CODE REVIEW

File: frontend/src/global_styles/content/work_packages/tabs/_files.sass [hunk 1]
```diff
@@ -132,6 +132,10 @@
       &:not(:last-child)
         margin-right: $spot-spacing-0-5
 
+  @media only screen and (max-width: 679px)
+    & &--file-list-item-floating-wrapper:hover &--file-list-item-floating-actions
```
Comment (HDinger) [inline frontend/src/global_styles/content/work_packages/tabs/_files.sass]: Just wondering: In which cases do I have a hover effect on a mobile device?

Comment (Kharonus) [reply frontend/src/global_styles/content/work_packages/tabs/_files.sass]: This is probably related to the focus. Tbh I wasn't able to test it yet, a pull preview might be handy in here.

The issue is this: On iOS currently you have to click twice on the attachment. First click somehow only opens the floating actions.

On android (brave Browser) the click was working fine, but when I use the back button, the focus is still set, so I can see the floating actions, too.

Comment (Kharonus) [reply frontend/src/global_styles/content/work_packages/tabs/_files.sass]: **UPDATE:** at least in the brave browser on Android, this override works fine. getting back via browser button still has the clicked file focused, but the floating actions are not visible.

Comment (Kharonus) [reply frontend/src/global_styles/content/work_packages/tabs/_files.sass]: **UPDATE 2**: @apfohl tested on iOS and there the behaviour is now the same (only small difference, the focus does not render a border there, but this is unrelated).

@HDinger Do you have any further question? Or do you want to check it on your mobile once perhaps? pullpreview is available.

Comment (HDinger) [reply frontend/src/global_styles/content/work_packages/tabs/_files.sass]: I will probably need help to connect a file storage to the PullPreview Instance, as I could not manage. 🙈

**Review by HDinger — COMMENTED** [review]
We have a class called `hidden-for-mobile`. Without having it tested, I assume that you can add that to the `op-files-tab--file-list-item-floating-actions` element and it should do the trick without any of the other changes being necessary.

File: frontend/src/app/shared/components/attachments/attachment-list/attachment-list-item.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<div class="spot-list--item-floating-wrapper">
+<div class="spot-list--item-floating-wrapper op-files-tab--file-list-item-floating-wrapper">
```
Comment (HDinger) [inline frontend/src/app/shared/components/attachments/attachment-list/attachment-list-item.html]: That class is not needed any more, right?

**Review by HDinger — APPROVED** [review]
Looks fine 👍 I just have a minor remark, otherwise good to merge.

File: frontend/src/app/shared/components/attachments/attachment-list/attachment-list-item.html (hunk 2 above)
Comment (Kharonus) [reply frontend/src/app/shared/components/attachments/attachment-list/attachment-list-item.html]: will double check. I believe there are some stylings based on this class in `file-link-list`. If I cannot find a good reason for to keep it, I'll remove it and merge afterwards. 

Thanks

**Pull request merged**